openmc/data/fission_energy.py

```diff
@@ -0,0 +1,627 @@
+
+    Parameters
+    ----------
+    endf_files : Collection of str
+        Strings giving the paths to the ENDF files that will be parsed for data.
+    output_name : str
+        Name of the output HDF5 file.  Default is 'fission_Q_data.h5'.
+    comment : str
+        Comment to write in the output HDF5 file.  Defaults to no comment.
+    verbose : bool
+        If True, print the name of each isomer as it is read.  Defaults to
+        False.
+
+    """
+    # Open the output file.
+    if output_name is None: output_name = 'fission_Q_data.h5'
+    out = h5py.File(output_name, 'w', libver='latest')
+
+    # Write comments, if given.  This commented out comment is the one used for
+    # the library distributed with OpenMC.
+    #comment = ('This data is extracted from ENDF/B-VII.1 library.  Thanks '
+    #           'evaluators, for all your hard work :)  Citation:  '
+    #           'M. B. Chadwick, M. Herman, P. Oblozinsky, '
+    #           'M. E. Dunn, Y. Danon, A. C. Kahler, D. L. Smith, '
+    #           'B. Pritychenko, G. Arbanas, R. Arcilla, R. Brewer, '
+    #           'D. A. Brown, R. Capote, A. D. Carlson, Y. S. Cho, H. Derrien, '
+    #           'K. Guber, G. M. Hale, S. Hoblit, S. Holloway, T. D. Johnson, '
+    #           'T. Kawano, B. C. Kiedrowski, H. Kim, S. Kunieda, '
+    #           'N. M. Larson, L. Leal, J. P. Lestone, R. C. Little, '
+    #           'E. A. McCutchan, R. E. MacFarlane, M. MacInnes, '
+    #           'C. M. Mattoon, R. D. McKnight, S. F. Mughabghab, '
+    #           'G. P. A. Nobre, G. Palmiotti, A. Palumbo, M. T. Pigni, '
+    #           'V. G. Pronyaev, R. O. Sayer, A. A. Sonzogni, N. C. Summers, '
+    #           'P. Talou, I. J. Thompson, A. Trkov, R. L. Vogt, '
+    #           'S. C. van der Marck, A. Wallner, M. C. White, D. Wiarda, '
+    #           'and P. G. Young. ENDF/B-VII.1 nuclear data for science and '
+    #           'technology: Cross sections, covariances, fission product '
+    #           'yields and decay data", Nuclear Data Sheets, '
+    #           '112(12):2887-2996 (2011).')
+    if comment is not None:
+        out.attrs['comment'] = np.string_(comment)
+
+    # Declare the order of the components.  Use fixed-length numpy strings
+    # because they work well with h5py.
+    labels = np.array(('EFR', 'ENP', 'END', 'EGP', 'EGD', 'EB', 'ENU', 'ER',
+                       'ET'), dtype='S3')
+    out.attrs['component order'] = labels
+
+    # Iterate over the given files.
+    if verbose: print('Reading ENDF files:')
+    for fname in endf_files:
+        if verbose: print(fname)
+
+        ident = identify_nuclide(fname)
+
+        # Skip non-fissionable nuclides.
+        if not ident['LFI']: continue
+
+        # Get the important bits.
+        data = _extract_458_data(fname)
+        if data is None: continue
+        value, uncertainty = data
+
+        # Make a group for this isomer.
+        name = str(ident['Z']) + str(ident['A'])
```

Comment:
Maybe `ATOMIC_SYMBOL(ident['Z'])` instead? Would make it consistent with our other HDF5 data